win/pkgs/cuda.ps1

```diff
@@ -25,3 +25,26 @@ Write-Host "Installing CUDA..."
 & "${DownloadDir}/${exe}" -s | Out-Null
 
 ${Env:CUDA_PATH}=[System.Environment]::GetEnvironmentVariable("CUDA_PATH","Machine")
+
+# CUDA MSBuild integration is not working for VS BuildTools.
+# Manually install CUDA Build Customizations files into MSBuild customization folder.
+#
+# See Table 4. CUDA Visual Studio .props locations from
+# https://docs.nvidia.com/cuda/cuda-installation-guide-microsoft-windows/index.html
+$vs_where = "${Env:ProgramFiles(x86)}/Microsoft Visual Studio/Installer/vswhere.exe"
+if (Test-Path $vs_where)
+{
+    $vs_home = & $vs_where -latest -products 'Microsoft.VisualStudio.Product.BuildTools' -property installationPath
+    if ($vs_home)
+    {
+        Write-Host "Found existing VS installtaion under `"$vs_home`". Skip installation."
+
+        $cuda_vsext_dir = "${Env:CUDA_PATH}/extras/visual_studio_integration/MSBuildExtensions"
+        $msbuild_custom_dir = "$vs_home/MSBuild/Microsoft/VC/v160/BuildCustomizations"
```

Comment:
Don't hard code v160 here.

Reply:
This is required from CUDA documentation:
> Visual Studio 2019 | C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\MSBuild\Microsoft\VC\v160\BuildCustomizations

Roaster script is currently hard coding VS2019 as the build tool.

Reply:
We don't really hard-code it.
We provide a single-place config as env var, so it is more like a config in code.
Adding this kind of assumption in random places with turn that into "hard-code" and make future update much harder.

You should probably list the dir and pick the highest version if multiple.

Reply:
Patch every version which has BuildCustomization folder.